scarb/src/ops/subcommands.rs

```diff
@@ -1,14 +1,87 @@
-use std::env;
+use std::collections::BTreeMap;
 use std::ffi::OsStr;
+use std::fmt::Display;
 use std::path::PathBuf;
 use std::process::Command;
+use std::{env, fs};
 
 use anyhow::{bail, Result};
 
 use crate::core::Config;
 use crate::process::{exec_replace, is_executable};
 use crate::SCARB_ENV;
 
+pub const EXTERNAL_CMD_PREFIX: &str = "scarb-";
```

Comment:
hmm this makes this constant available as `scarb::ops::EXTERNAL_CMD_PREFIX` which doesn't sound right

I think you could move this to `src/lib.rs` along other constatnts